src/lein_tools_deps/plugin.clj

```diff
@@ -79,7 +79,7 @@
   (-> deps-files
       reader/read-deps))
 
-(defmulti leinize (fn [[dep-key dep-val]]
+(defmulti leinize (fn [[_ dep-val]]
```

Comment:
@HughPowell I think here it is useful to know the name even though it's ignored... i.e. it makes it clear these are `[key value]` pairs (from `seq`ing a hashmap).

Elsewhere we use the convention `_arg-name`, to provide the hint.  IIRC IDE's respect this as well.

Reply:
Can fix this up later though.

Reply:
Yep, didn't know about _arg-name, but looks like a really good idea.